lib/index.js

```diff
@@ -351,7 +352,7 @@ class Mcdev {
      */
     static async deleteByKey(businessUnit, type, customerKey) {
         Util.logger.info('mcdev:: delete');
-        properties = properties || File.loadConfigFile();
+        const properties = await config.getProperties();
```

Comment:
depending on how you want to see it either this or cli.js is highest level of execution. neither are handling potential errors returned by your new method at this point. hard no!